### PR TITLE
Update redis-service-configuration, add clarification note regarding ece-tools and port.

### PR DESCRIPTION
**Pull request by perezalo**
When Platform Team (P.sh) add new redis-session service to an environment, they are not able to guarantee the port where redis-session is listening will be the same for all environments in a project. If we "hardcode" the redis-session port in the '.magento.env.yaml' config file we could provoque an error in some other environments of the project, if the redis-session service is not listening at that port. In this "Note", we clarify that only is needed to add the "redis.port" configuration section, if "ece-tools" are not able to get the port from RELATIONSHIPS services info environment variable (provided by Platform Team into the systems).

## Purpose of this pull request

This pull request (PR) is for adding a clarification NOTE into the Doc.

## Affected pages

- https://experienceleague.adobe.com/en/docs/commerce-operations/implementation-playbook/best-practices/planning/redis-service-configuration

If you are fixing a GitHub issue, using the GitHub keyword format (https://help.github.com/en/articles/closing-issues-using-keywords#closing-an-issue-in-a-different-repository) closes the issue when this pull request is merged. Example: `Fixes #1234`.

`main` is the default branch. Merged pull requests to `main` go live on the site automatically. Any requested changes to content on the `main` branch must be related to the released codebase. Any content related to future releases goes in the `develop` branch.

See Contribution guidelines (https://github.com/AdobeDocs/commerce-operations.en/blob/main/contributing.md) for more information.
-->

### CODE REVIEW

**Comment by devops-devdocs** [comment]
Can one of the admins verify this patch?

**Review by perezalo — COMMENTED** [review]
Please, could you review this PR??!!

File: help/implementation-playbook/best-practices/planning/redis-service-configuration.md [hunk 1]
```diff
@@ -200,6 +200,10 @@ Separating the Redis cache from Redis session allows you to manage the cache and
 
 1. Add the port number to the `.magento.env.yaml` configuration file.
 
+   >[!NOTE]
```
Comment (danidelcar) [inline help/implementation-playbook/best-practices/planning/redis-service-configuration.md:207]:
```suggestion
   >[!NOTE]
   > Configure the redis session port only if `ece-tools` is unable to automatically detect it from the `MAGENTO_CLOUD_RELATIONSHIPS` redis session service definition.
```

File: help/implementation-playbook/best-practices/planning/redis-service-configuration.md [hunk 2]
```diff
@@ -200,6 +200,10 @@ Separating the Redis cache from Redis session allows you to manage the cache and
...
+   >Set the redis session port, is only needed if `ece-tools` did not get it correctly from `MAGENTO_CLOUD_RELATIONSHIPS` redis-session service definition.
```
Comment (danidelcar) [inline help/implementation-playbook/best-practices/planning/redis-service-configuration.md]:
```suggestion
```

If previous comment is applied, remove the rest of the entry

File: help/implementation-playbook/best-practices/planning/redis-service-configuration.md [hunk 3]
```diff
@@ -200,6 +200,10 @@ Separating the Redis cache from Redis session allows you to manage the cache and
...
+   >
```
Comment (danidelcar) [inline help/implementation-playbook/best-practices/planning/redis-service-configuration.md]:
```suggestion
 ```

**Review by danidelcar — APPROVED** [review]
approved, with a suggestion

**Review by jeff-matthews — APPROVED** [review]

**Comment by perezalo** [comment]
Ok, in the end... we have the right NOTE :-D

**Pull request closed without merging**